var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -105,7 +105,7 @@ def setup_dependent_run_environment(self, env, dependent_spec):
         self.setup_run_environment(env)
 
 
-class BaseBuilder:
+class BaseBuilder(metaclass=spack.builder.PhaseCallbacksMeta):
```

Comment:
I think:
```python
import spack.builder
```
is missing

Reply:
Well, it works and the style checks seem to pass. Should I still add the `import`?

Reply:
It would be better, since it works by chance due to Python `sys.modules` state